improver.py

```diff
@@ -0,0 +1,75 @@
+# Written by Goncalopp and RedFantom
+# Thranta Squadron GSF CombatLog Parser, Copyright (C) 2016 by Goncalopp and RedFantom
+# All additions are under the copyright of their respective authors
+# For license see LICENSE
+from PIL import ImageEnhance, Image, ImageOps
+import files
+import numpy
+import cv2
+
+
+class ImageImprover(object):
+    """
+    Improver class that provides a wrap around the ImageEnhance classes from PIL/pillow
+    The Improved image can be accessed through object.image, this is a PIL/pillow RGB
+    format image, that is NOT compatible with the cv2 based files.ImageFile class!
```

Comment:
Yes, that is the case. After the improvements have been applied, the image in the `ImageFile` object is altered so the improved image can be used with the OCR functions without altering the original file. It would be possible to allow the user to put in Pillow image files as well by doing instance checking, but I don't know whether that would actually make things easier as you then support a data format *as input* that isn't used in the whole rest of the code.